common/environment.go

```diff
@@ -190,7 +190,7 @@ func (EnvironmentVariable) CredentialType() EnvironmentVariable {
 func (EnvironmentVariable) DefaultServiceApiVersion() EnvironmentVariable {
 	return EnvironmentVariable{
 		Name:         "AZCOPY_DEFAULT_SERVICE_API_VERSION",
-		DefaultValue: "2018-03-28",
+		DefaultValue: "2018-11-09",
```

Comment:
TODO: Consider the implications of adjusting the default service API version

Reply:
It's necessary in this case, but I don't know if it'll cause any other problems that need to be fixed before this can ship.

Reply:
It seems fine to me, there are no changes to existing APIs, just additions.